lib/Doctrine/ORM/Tools/Console/EntityManagerProvider/ConnectionFromManagerProvider.php

```diff
@@ -11,14 +11,18 @@ final class ConnectionFromManagerProvider implements ConnectionProvider
     /** @var EntityManagerProvider */
     private $entityManagerProvider;
 
-    public function __construct(EntityManagerProvider $entityManagerProvider)
+    /** @var string */
+    private $defaultManagerName;
+
+    public function __construct(EntityManagerProvider $entityManagerProvider, $defaultManagerName = 'default')
     {
         $this->entityManagerProvider = $entityManagerProvider;
+        $this->defaultManagerName    = $defaultManagerName;
     }
 
     public function getDefaultConnection(): Connection
     {
-        return $this->entityManagerProvider->getManager('default')->getConnection();
+        return $this->entityManagerProvider->getManager($this->defaultManagerName)->getConnection();
```

Comment:
I wonder whether this should leverage `EntityManagerProvider::getDefaultManager()` now when we have it (if `$defaultManagerName` was not provided).

Reply:
Yes you are right, this would be better.